### PR TITLE
Deadlines added and fixed search issue

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var config = {
     storageBucket: "awesomeapp-4ec8d.appspot.com",
     messagingSenderId: "523185544926"
 };
-firebase.initializeApp(config);
+var fbase = firebase.initializeApp(config);
```
Comment (elharony) [inline js/app.js]: We don't need to assign the `firebase` initialization. We are only `initialize` its configuration to use its features.
Do you think that we need it for a particular case?

File: js/app.js [hunk 2]
```diff
@@ -206,6 +232,7 @@ function getStudents(containerElement, projectName, queryFlag) {
     .get()
     .then(function(querySnapshot) {
         containerElement.innerHTML = '';
+        if(queryFlag && search.value === '') return;
```
Comment (elharony) [inline js/app.js:259]: What about a nicer `if condition` using the shorthanded?
`(condition) ? // to be executed when true : to be executed when false`

File: js/app.js [hunk 3]
```diff
@@ -423,6 +451,45 @@ function getProjects(arrayOfData, containerElement, trackName) {
                     }
                 break;
             }
+
```
Comment (elharony) [inline js/app.js:478]: Awesome work! 👍

**Review by elharony — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 1 above)
Comment (ghost) [reply js/app.js]: Hi there! Yes you're right )) I just forgot to remove this assignment ))) I did it for testing purposes. I don't remember what I actually was going to do with it but now it is completely unnecessary. Fixed))

File: js/app.js (hunk 2 above)
Comment (ghost) [reply js/app.js:259]: The problem is that when we need to interrupt the script execution we need to use return statement in the function body. But we can't use return in shorthand for if-else.
This line is not valid
(queryFlag && search.value === '') ? return : '' ;

Comment (elharony) [reply js/app.js:259]: Got it. 👍

**Pull request merged**